bundles/org.pitest.pitclipse.launch.ui/src/org/pitest/pitclipse/launch/ui/PitMutatorsTab.java

```diff
@@ -0,0 +1,369 @@
+ * the License.
+ ******************************************************************************/
+
+package org.pitest.pitclipse.launch.ui;
+
+import static org.pitest.pitclipse.core.preferences.PitPreferences.INDIVIDUAL_MUTATORS;
+import static org.pitest.pitclipse.core.preferences.PitPreferences.MUTATORS;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.debug.core.ILaunchConfiguration;
+import org.eclipse.debug.core.ILaunchConfigurationWorkingCopy;
+import org.eclipse.debug.ui.AbstractLaunchConfigurationTab;
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.jface.layout.GridLayoutFactory;
+import org.eclipse.jface.viewers.ArrayContentProvider;
+import org.eclipse.jface.viewers.CellLabelProvider;
+import org.eclipse.jface.viewers.CheckboxTableViewer;
+import org.eclipse.jface.viewers.TableViewerColumn;
+import org.eclipse.jface.viewers.Viewer;
+import org.eclipse.jface.viewers.ViewerCell;
+import org.eclipse.jface.viewers.ViewerFilter;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.graphics.Font;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.program.Program;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Link;
+import org.pitest.mutationtest.engine.gregor.config.Mutator;
+import org.pitest.pitclipse.core.Mutators;
+import org.pitest.pitclipse.core.PitCoreActivator;
+import org.pitest.pitclipse.runner.config.PitConfiguration;
+import org.pitest.pitclipse.ui.utils.PitclipseUiUtils;
+
+/**
+ * Tab allowing to configure a PIT analyze.
+ * @author Jonas Kutscha
+ */
+public final class PitMutatorsTab extends AbstractLaunchConfigurationTab {
+    public static final String NAME = "Mutators";
+
+    private static final int NUMBER_OF_COLUMNS = Mutators.getMainGroup().size() + 2;
+    private static final String DESCRIPTION_TEXT = "Select the mutators used to alter the code.";
+    private static final String MUTATOR_LINK_TEXT = "See the documentation on Pitest.org";
+    private static final String MUTATOR_LINK = "https://pitest.org/quickstart/mutators/";
+    public static final String CUSTOM_MUTATOR_RADIO_TEXT = "Mutators selected below";
+    /**
+     * Needed for {@link ButtonSelectionListener#widgetSelected(SelectionEvent)}
+     */
+    private static final String CUSTOM_MUTATOR_RADIO_DATA = "CUSTOM";
+    private static final String COLUMN_DESCRIPTION = "Description";
+    private static final String COLUMN_NAME = "Name";
+    private static final String NO_DESCRIPTION_TEXT = "No description found yet.";
+    private static final String ERROR_MESSAGE = "At least one mutator or mutator group needs to be selected!";
+    private String mutators;
```

Comment:
This variable confuses me as well. You initialize it from the `MUTATORS` preference but it actually stores the current mutation group, right? If so, could you rename it accordingly (`currentMutatorGroup`)?